docs/smart-contracts/tokens/wayfinding.mdx

```diff
@@ -0,0 +1,66 @@
+---
+sidebar_position: 5
+title: Tokens Overview
+description: Choose between the Stellar Asset Contract and Custom Tokens
+---
+
+Tokens exist in two forms on Stellar: 1) assets issued by Stellar accounts (Stellar assets can have a [Stellar Asset Contract (SAC)](./stellar-asset-contract.mdx) implementation), and 2) as a [custom token](./token-interface.mdx) created by a smart contract.
+
+Several factors (such as project-specific requirements and network capabilities) can help you determine whether to issue an asset on Stellar or create a custom token with a smart contract for your project.
+
+However:
+
+### TL;DR
+
+If possible, we recommend issuing a Stellar asset and (if needed) using the SAC to interact with that asset in smart contracts. More on why below.
+
+## Issuing assets on Stellar
+
+Stellar is first-class in asset tokenization — issuing an asset can be done in [just one single transaction](https://developers.stellar.org/docs).
```

Comment:
Is the goal to say Stellar is the best, or that Stellar has first-class support for assets? I think at the moment this does a bit of both, but maybe stronger with the below.

Also, one transaction to issue a asset is maybe a little misleading. In most cases you need to set things up, but I think the fact is that you don't have to write a contract.

I don't feel strongly about this if you prefer the existing wording here.

```suggestion
Stellar has first-class support for asset tokenization — issuing an asset can be done using a [built-in transaction](https://developers.stellar.org/docs) without the development of a smart contract.
```